test/e2e/kafka_source_test.go

```diff
@@ -97,14 +214,80 @@ func testKafkaSource(t *testing.T, name string, version string, messageKey strin
 
 	client.WaitForAllTestResourcesReadyOrFail(context.Background())
 
-	helpers.MustPublishKafkaMessage(client, kafkaBootstrapUrl, kafkaTopicName, messageKey, messageHeaders, messagePayload)
+	helpers.MustPublishKafkaMessage(client, kafkaBootstrapUrlPlain, kafkaTopicName, messageKey, messageHeaders, messagePayload)
 
 	eventTracker.AssertExact(1, recordevents.MatchEvent(matcherGen(cloudEventsSourceName, cloudEventsEventType)))
 }
 
 func TestKafkaSource(t *testing.T) {
 	time, _ := cetypes.ParseTime("2018-04-05T17:31:00Z")
 
+	auths := map[string]struct {
+		auth authSetup
+	}{
+		"plain": {
+			auth: authSetup{
+				bootStrapServer: kafkaBootstrapUrlPlain,
+				Net: AdapterNet{
+					SASL: AdapterSASL{
+						Enable: false,
+					},
+					TLS: AdapterTLS{
+						Enable: false,
+					},
+				},
+			},
+		},
+		"sasl-512": {
+			auth: authSetup{
+				bootStrapServer: kafkaBootstrapURLSASL,
+				Net: AdapterNet{
+					AdapterSASL{
+						Enable:   true,
+						User:     kafkaClusterSASLUsername,
+						Password: kafkaClusterPassword,
+						Type:     "SCRAM-SHA-512",
+					},
+					AdapterTLS{
+						Enable: false,
```

Comment:
we have SASL only enabled w TLS:
https://github.com/knative-sandbox/eventing-kafka/blob/master/test/config/100-kafka-ephemeral-triple-2.6.0.yaml#L20-L25